tlsr9/common/b9x_sleep.h

```diff
@@ -33,6 +33,6 @@
 bool b9x_suspend(uint32_t wake_stimer_tick);
 #if CONFIG_SOC_SERIES_RISCV_TELINK_B9X_RETENTION
 bool b9x_deep_sleep(uint32_t wake_stimer_tick);
-#endif /* CONFIG_SOC_SERIES_RISCV_TELINK_B9X_RETENTION */
+#endif /* (CONFIG_SOC_SERIES_RISCV_TELINK_B9X_RETENTION */
```

Comment:
revert 
use original 
`#endif /* CONFIG_SOC_SERIES_RISCV_TELINK_B9X_RETENTION */`

Reply:
renamed b9x_sleep.h to tl_sleep.h for both `b9x` and `tlx` in latest commit 0941ddd.